app/policies/work_policy.rb

```diff
@@ -21,6 +21,14 @@ def review?
     collection_reviewer? || collection_manager?
   end
 
+  scope_for :relation, :collection do |relation, collection:|
```

Comment:
Since we're likely to be primarily using AR scopes, this could be shortened to:

```suggestion
  relation_scope(:collection) do |relation, collection:|
```

https://actionpolicy.evilmartians.io/#/scoping?id=active-record-scopes